guides/common/modules/proc_installing-and-configuring-puppet-agent-during-host-registration.adoc

```diff
@@ -46,11 +46,20 @@ endif::[]
 . In the {ProjectWebUI}, navigate to *Configure* > *Global Parameters* to add host parameters globally.
 Alternatively, you can navigate to *Configure* > *Host Groups* and edit or create a host group to add host parameters only to a host group.
 . Enable the Puppet agent using a host parameter in global parameters or a host group.
-ifdef::katello,orcharhino,satellite[]
+// consolidate with line 54 as soon as Satellite client contains Puppet agent 8
+ifdef::satellite[]
++
 Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
 endif::[]
-ifndef::katello,orcharhino,satellite[]
-Add a host parameter named `enable-official-puppet7-repo`, select the *boolean* type, and set the value to `true`.
+ifdef::katello,orcharhino[]
++
+* To use Puppet 8, add a host parameter named `enable-puppet8`, select the *boolean* type, and set the value to `true`.
```

Comment:
The whole parameter `enable-puppet8` doesn't exist within Foreman. This is IMHO a bug so I opened https://projects.theforeman.org/issues/37891 to track this.

At this point we should really change the implementation to be version independent. It's really AIO packaging vs non-AIO. I'd even argue that these days the AIO packaging is more common and we should default to it.